lib/Elastica/Transport/Guzzle.php

```diff
@@ -93,6 +94,7 @@ public function exec(Request $request, array $params)
             [
                 'request_header' => $request->getMethod(),
                 'http_code' => $res->getStatusCode(),
+                'body' => $responseBody,
```

Comment:
I wonder if this could have some side effects?

Reply:
No idea – it's my first contribution to this package. Don't think anything in FOSElastica is trying to read this (if i remember correctly)

Reply:
@ruflin it should break anything, i'm doing some tests on that and then I let u know.

Reply:
@p365labs Not sure I can follow. So this is good?

In general it's just a gut feeling I have that the above could impact other things. If we don't find something I'm good with moving forward.

Reply:
Thanks for looking into it both – happy to make any changes when needed

Reply:
Ha, you're right – i'm doing something completely stupid here!

The use-case was to pass the body to have useful error messages (in case the AwsAuth is passed incorrect credentials) in FOSElasticaBundle

Reply:
Does this mean we get a follow up PR?

Reply:
@jandom if I got it correctly, you're using the `body` to display the response received from the client. In here: https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1465/files/e5386eac9a4b1ee22d7d215ebe32afc5fec046af#diff-dbbc2aef2d5d1dd159ad264fb54364dcL51, why don't you keep the response data and use it in case of an exception?

Reply:
The ` $responseData = $response->getData();` can be used instead of storing twice the body contents.